index.js

```diff
@@ -80,8 +130,28 @@ genderCount(data);
 
 const promo20 = users => {
   // Your code goes here ...
+  const qualify = [];
+
+  //get users with balance above 20000 and push to qualify array
+  for(let user of users){
+    let str = user.balance
+    //convert string to number
+    let num = parseInt(str.replace(/[^\d\.\-]/g, ""))
+    //check if greater than 20000
```

Comment:
I love the fact you commented at least the regex. I am not profiecient at Regex and whenever I have to use them I try to always comment so I know exactly what I am looking at

Reply:
😃